ods_ci/tests/Tests/1000__model_serving/1006__model_serving_ovms_on_kserve_cli.robot

```diff
@@ -0,0 +1,111 @@
+*** Settings ***
+Documentation     Collection of CLI tests to validate the model serving stack for different Large Language Models (LLM).
+...               These tests leverage on TGIS Standalone Serving Runtime
+Resource          ../../Resources/OCP.resource
+Resource          ../../Resources/CLI/ModelServing/llm.resource
+Library           OpenShiftLibrary
+Suite Setup       Suite Setup
+Suite Teardown    Suite Teardown
+Test Tags         KServe-OVMS
+
+
+*** Variables ***
+${TEST_NS}=        ovmsmodel
+${RUNTIME_NAME}=  ovms-runtime   
+${USE_PVC}=    ${TRUE}
+${DOWNLOAD_IN_PVC}=    ${TRUE}
+${USE_GPU}=    ${FALSE}
+${KSERVE_MODE}=    Serverless    #RawDeployment   #Serverless
+${MODEL_FORMAT}=   onnx
+${PROTOCOL}=     http
+${OVERLAY}=      ${EMPTY}
+${MODELS_BUCKET}=    ${S3.BUCKET_1}
+${INFERENCE_INPUT}=    @tests/Resources/Files/modelmesh-mnist-input.json
+${INFERENCE_INPUT_OPENVINO}=    @tests/Resources/Files/openvino-example-input.json
+#${EXPECTED_INFERENCE_SECURED_OUTPUT}=    {"model_name":"${MODEL_NAME}__isvc-83d6fab7bd","model_version":"1","outputs":[{"name":"Plus214_Output_0","datatype":"FP32","shape":[1,10],"data":[-8.233053,-7.7497034,-3.4236815,12.3630295,-12.079103,17.266596,-10.570976,0.7130762,3.321715,1.3621228]}]}  #robocop: disable
+#${EXPECTED_INFERENCE_OUTPUT_OPENVINO}=    {"model_name":"${MODEL_NAME}__isvc-8655dc7979","model_version":"1","outputs":[{"name":"Func/StatefulPartitionedCall/output/_13:0","datatype":"FP32","shape":[1,1],"data":[0.99999994]}]}  #robocop: disable
+
+*** Test Cases ***
+Verify User Can Serve And Query ovms Model
+    [Documentation]    Basic tests for preparing, deploying and querying model
+    ...                using Kserve and ovms runtime
+    [Tags]    OVMS
+    ...       Tier1
+    ...       Smoke
+    ...       OpenDataHub
+    ...       RHOAIENG-9045
+    Setup Test Variables    model_name=test-dir    use_pvc=${USE_PVC}    use_gpu=${USE_GPU}
+    ...    kserve_mode=${KSERVE_MODE}
+    ${EXPECTED_INFERENCE_SECURED_OUTPUT}  Set Variable    {"model_name":"${model_name}__isvc-83d6fab7bd","model_version":"1","outputs":[{"name":"Plus214_Output_0","datatype":"FP32","shape":[1,10],"data":[-8.233053,-7.7497034,-3.4236815,12.3630295,-12.079103,17.266596,-10.570976,0.7130762,3.321715,1.3621228]}]}  #robocop: disable
```

Comment:
If you need this set it in the Variables section instead of the test

Reply:
if i set in in the variable it will fail as model_name is defined in the later phase of the test